tests/test_scrape.py

```diff
@@ -7,6 +7,17 @@
 
 class ScrapeTest(unittest.TestCase):
 
+    def setUp(self):
+        self.json = {'data' : {'children' : [{'data' : {'preview' : {
+                          'images' : [{'source' : {'url' :
+                                'url.com/some_url.png?21'}}]}}}]}}
```

Comment:
Line is longer than allowed. (83 > 79)

*LineLengthBear, severity NORMAL, section `linelength`.*